ds_judgements_public_ui/sass/includes/_judgment_text.scss

```diff
@@ -170,6 +170,18 @@
   &__approved-judgment {
     text-align: center;
   }
+
+  &__pr-center {
```

Comment:
What does pr stand for?

Reply:
aha, my thinking here was that it'd be useful to somehow give all purely presentational classes a common prefix so that we can identify them later in the event that we find a way of getting rid of them, hence `pr-` for `presentation`. I've added a comment to explain this (as it's definitely not clear on its own), but to be honest, I'm not actually convinced it's necessary, thinking again, so could be convinced to drop them!